recipes/ssu-align/post-link.sh

```diff
@@ -0,0 +1,22 @@
+echo "
+ssu-align has been installed.  To run it properly please set the
+environment variable SSUALIGNDIR using one the of the following
```

Comment:
@mworkentine why is this needed? Can this program not simply use the pwd from the the binary?

Reply:
SSUAlign is weird and uses that ENVVAR to find libraries instead of the binary path since the install lets you install them to different places if you want

Reply:
Can we include a very tiny bash wrapper for both binaries that do the export and forward all parameters to the original binary. So something like:

* rename original binary to PREFIX/bin/_ssu-align
* create a wrapper called PREFIX/bin/ssu-align that calles _ssu-align but exports the ENV before